README.md

```diff
@@ -10,7 +10,45 @@ Install with [npm](http://github.com/isaacs/npm):
 
 A pre requisite it to have [openssl](http://www.openssl.org/) installed and its /bin to be on the system path. I used version 1.0.1c but it should work on older versions too.
 
+## Supported Algorithms
+
+### Canonicalization and Transformation Algorithms
+
+* Exclusive Canonicalization http://www.w3.org/2001/10/xml-exc-c14n#
+* Exclusive Canonicalization with comments http://www.w3.org/2001/10/xml-exc-c14n#WithComments
+* Enveloped Signature transform http://www.w3.org/2000/09/xmldsig#enveloped-signature
+
+### Hashing Algorithms
+
+* SHA1 digests http://www.w3.org/2000/09/xmldsig#sha1
+* SHA256 digests http://www.w3.org/2001/04/xmlenc#sha256
+
+### Signature Algorithms
+
+* RSA-SHA1 http://www.w3.org/2000/09/xmldsig#rsa-sha1
+* RSA-SHA256 http://www.w3.org/2001/04/xmldsig-more#rsa-sha256
+* RSA-SHA512 http://www.w3.org/2001/04/xmldsig-more#rsa-sha512
+
+by default the following algorithms are used:
+
+*Canonicalization/Transformation Algorithm:* Exclusive Canonicalization http://www.w3.org/2001/10/xml-exc-c14n#
+
+*Hashing Algorithm:* SHA1 digest http://www.w3.org/2000/09/xmldsig#sha1
+
+*Signature Algorithm:* RSA-SHA1 http://www.w3.org/2000/09/xmldsig#rsa-sha1
+
+[You are able to extend xml-crypto with further algorithms.](#customizing-algorithms)

```

Comment:
Should this say "You are able to extend xml-crypto with custom algorithms" or maybe "additional algorithms"